examples/verif-manual/vm-011-residual-stress-problem.py

```diff
@@ -0,0 +1,174 @@
+r""".. _ref_vm11_example:
+
+Residual Stress Problem
+----------------------------
+Problem Description:
+ - A chain hoist is attached to the ceiling through three tie rods as shown below.
+   The tie rods are made of cold-rolled steel with yield strength :math:`\sigma_{yp}`
+   and each has an area A. Find the deflection :math:`\delta` at load :math:`F_1` when
+   the deflections are elastic in all three rods. When the frame is loaded to :math:`F_2`
+   (where all three rods become fully plastic), and then unloaded,
+   find the residual stress :math:`\sigma_r` in the central rod.
+
+Reference:
+ - S. H. Crandall, N. C. Dahl, *An Introduction to the Mechanics of Solids*,
+   McGraw-Hill Book Co., Inc., New York, NY, 1959, pg. 234, problem 5.31.
+
+Analysis Type(s):
+ - Static Analysis (``ANTYPE = 0``)
+
+Element Type(s):
+ - 3-D Spar (or Truss) Elements (``LINK180``)
+
+.. figure:: ../_static/vm11_setup_1.png
+    :align: center
+    :width: 400
+    :alt:  VM11 Problem Sketch
+    :figclass: align-center
+
+    **VM11 Problem model**
+
+Material Properties
+ - :math:`\sigma_{yp} = 30,000\,psi`
+ - :math:`E = 30 \cdot 10^6\,psi`
+
+.. figure:: ../_static/vm11_setup_2.png
+    :align: center
+    :width: 400
+    :alt:  VM11 Material Model
+    :figclass: align-center
+
+    **VM11 Material Model**
+
+Geometric Properties:
+ - :math:`A = 1\,in^2`
+ - :math:`l = 100\,in`
+ - :math:`\Theta = 30°`
+
+Loading:
+ - :math:`F_1 = 51,961.5\,lb`
+ - :math:`F_2 = 81,961.5\,lb`
+
+Analysis Assumptions and Modeling Notes:
+ - Automatic load stepping (:meth: Mapl.autots <ansys.mapdl.core.Mapdl.autots>,ON)
+   is used to obtain the nonlinear plastic solution (load steps 2 and 3).
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm11_setup.png'
+
+import math
+
+from ansys.mapdl.core import launch_mapdl
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL.
+
+mapdl = launch_mapdl()
+mapdl.clear()  # optional as MAPDL just started
+
+###############################################################################
+# Pre-processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.verify("vm11")
+mapdl.prep7()
+mapdl.title("VM11 RESIDUAL STRESS PROBLEM")
+
+###############################################################################
+# Define element type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type ``LINK180``.
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Element type: LINK180.
+mapdl.et(1, "LINK180")
+mapdl.sectype(1, "LINK")
+mapdl.secdata(1)
+mapdl.mp("EX", 1, 30e6)
+mapdl.tb("PLAS", 1, "", "", "BKIN")  # TABLE FOR BILINEAR KINEMATIC HARDENING
```

Comment:
It might be interesting to add the name of the parameters instead of using `""` for the functions containing empty parameters

```suggestion
mapdl.tb(lab="PLAS", mat=1, tbopt="BKIN")  # TABLE FOR BILINEAR KINEMATIC HARDENING
```

Here is the corresponding [documentation](https://mapdl.docs.pyansys.com/version/stable/mapdl_commands/prep7/_autosummary/ansys.mapdl.core.Mapdl.tb.html#tb) for the function `tb`.